src/dialogs.c

```diff
@@ -841,8 +827,8 @@ on_font_dialog_response(GtkDialog *dialog, gint response, gpointer user_data)
 		{
 			gchar *fontname;
 
-			fontname = gtk_font_selection_dialog_get_font_name(
-				GTK_FONT_SELECTION_DIALOG(ui_widgets.open_fontsel));
+			fontname = gtk_font_chooser_get_font(
+				GTK_FONT_CHOOSER(ui_widgets.open_fontsel));
```

Comment:
`GTK_FONT_CHOOSER_DIALOG()`

Reply:
I think I did it right. Notice the original code - if first defines
```
#define GTK_FONT_SELECTION_DIALOG GTK_FONT_CHOOSER_DIALOG
```
but then, when it's called, there's an extra `GTK_FONT_CHOOSER()` which overrides it

Reply:
Okay, I didnt see the calls inside the macros.

But still, they are constructed with `gtk_font_chooser_dialog_new()` so I would think `GTK_FONT_CHOOSER_DIALOG()` is correct anyway